exercises/nucleotide-count/canonical-data.json

```diff
@@ -50,6 +50,17 @@
           }
         },
         {
+          "description": "is case-agnostic against a strand of mixed-case nucleotides",
+          "property": "nucleotideCounts",
+          "strand": "aGcTtTtCaTtCtGaCtGcAaCgGgCaAtAtGtCtCtGtGtGgAtTaAaAaAaGaGtGtCtGaTaGcAgC",
```

Comment:
Would `AaCcGgTt` also be valid for this test?

Is there a reason why this needs to be so long and mixed up?
If I'm debugging my program I'd like to be able to manually verify the expected inputs and outputs.

Reply:
@Insti
The input was taken directly from the original case above ("strand with multiple nucleotides").
I wanted to keep it consistent.
Any other suggested input is welcomed as well.

Reply:
I also object to the case above, but wasn't asking you to change that one.